toxcore/group_chats.c

```diff
@@ -89,14 +86,14 @@ typedef enum Group_Handshake_Request_Type {
 
 
 static bool group_number_valid(const GC_Session *c, int group_number);
-static int peer_add(Messenger *m, int group_number, const IP_Port *ipp, const uint8_t *public_key);
+static int peer_add(const Messenger *m, int group_number, const IP_Port *ipp, const uint8_t *public_key);
 static int peer_update(Messenger *m, int group_number, GC_GroupPeer *peer, uint32_t peer_number);
 static int group_delete(GC_Session *c, GC_Chat *chat);
 static void group_cleanup(GC_Session *c, GC_Chat *chat);
 static int get_nick_peer_number(const GC_Chat *chat, const uint8_t *nick, uint16_t length);
 static bool group_exists(const GC_Session *c, const uint8_t *chat_id);
 static int save_tcp_relay(GC_Connection *gconn, Node_format *node);
-int gcc_copy_tcp_relay(GC_Connection *gconn, Node_format *node);
+int gc_copy_tcp_relay(GC_Connection *gconn, Node_format *node);
```

Comment:
What is the naming convention? What does gcc stand for and what does gc stand for?

Reply:
gc -> groupchat, gcc -> groupchat connection, gca -> groupchat announce

All the gcc functions reside in group_connection, all the gc functions reside in group_chats, and all the gca functions reside in group_announce.